### PR TITLE
feat: add third party scanners page

### PR DESCRIPTION
**Pull request by McPatate**

### CODE REVIEW

File: docs/hub/security.md [hunk 1]
```diff
@@ -20,4 +20,5 @@ For any other security questions, please feel free to send us an email at securi
 - [Malware Scanning](./security-malware)
 - [Pickle Scanning](./security-pickle)
 - [Secrets Scanning](./security-secrets)
+- [3rd party scanners](./third-party-scanners)
```
Comment (julien-c) [inline docs/hub/security.md:23]:
```suggestion
- [3rd party scanners](./security-third-party-scanners)
```

File: docs/hub/third-party-scanners.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
+# 3rd Party scanners
```
Comment (julien-c) [inline docs/hub/third-party-scanners.md:1]:
```suggestion
# Third-party scanners
```

File: docs/hub/security-third-party-scanners.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+<img class="block" src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/hub/third-party-scans-list.png"/>
+
+Here is an example repository you can check out to see the feature in action: [mcpotato/42-eicar-street](https://huggingface.co/mcpotato/42-eicar-street).
+
+## Model security refresher
+
+To share models, we serialize the data structures we use to interact with the models, in order to facilitate storage and transport. Some serialization formats are vulnerable to nasty exploits, such as arbitrary code execution (looking at you pickle), making sharing models potentially dangerous.
+
+As Hugging Face has become the de facto platform for model sharing, we’d like to protect the community from this, hence why we have developed tools like [picklescan](https://github.com/mmaitre314/picklescan) and why we integrate 3rd party scanners.
```
Comment (julien-c) [inline docs/hub/security-third-party-scanners.md:19]:
```suggestion
As Hugging Face has become a popular platform for model sharing, we’d like to protect the community from this, hence why we have developed tools like [picklescan](https://github.com/mmaitre314/picklescan) and why we integrate 3rd party scanners.
```

File: docs/hub/security-third-party-scanners.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+
+Pickle is not the only exploitable format out there, [see for reference](https://github.com/Azure/counterfit/wiki/Abusing-ML-model-file-formats-to-create-malware-on-AI-systems:-A-proof-of-concept) how one can exploit Keras Lambda layers to achieve arbitrary code execution.
+
+## Protect AI's Guardian
+
+[Protect AI](https://protectai.com/)'s [Guardian](https://protectai.com/guardian) catches both pickle and Keras exploits. Guardian also benefits from reports sent in by their community of bounty [Huntr](https://huntr.com/)s.
```
Comment (julien-c) [inline docs/hub/security-third-party-scanners.md:25]: move that to the top

File: docs/hub/security-third-party-scanners.md [hunk 5]
```diff
@@ -0,0 +1,27 @@
+# Third-party scanners
```
Comment (julien-c) [inline docs/hub/security-third-party-scanners.md:1]:
```suggestion
# Third-party scanner: ProtectAI
```

**Review by seanpmorgan — CHANGES_REQUESTED** [review]

File: docs/hub/security-third-party-scanner-protect-ai.md [hunk 6]
```diff
@@ -0,0 +1,26 @@
+# Third-party scanner: Protect AI
+
+<Tip>
+Interested in joining our security partnership / providing scanning information on the Hub? Please get in touch with us over at security@huggingface.co.*
+</Tip>
+
+[Protect AI](https://protectai.com/)'s [Guardian](https://protectai.com/guardian) catches pickle, Keras, and other exploits as detailed on their [Knowledge Base page](https://protectai.com/insights/knowledge-base/). Guardian also benefits from reports sent in by their community of bounty [Huntr](https://huntr.com/)s.
+
+<!-- insert image of report -->
```
Comment (julien-c) [inline docs/hub/security-third-party-scanner-protect-ai.md:9]: @McPatate

**Review by julien-c — APPROVED** [review]

**Pull request merged**